modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -51,14 +53,18 @@ occ config:app:set federation auto_accept_trusted --value '0'
 occ config:app:set federatedfilesharing auto_accept_trusted --value 'yes'
 ----
 
+== Working With Proxies
+
+If the instance needs to go through a proxy, in `config/config.php`, set the `proxy` and `proxyuserpwd` configuration variables.
```

Comment:
I would like to have it better explained. There are oc instances that are not connected to the internet. They have no possibility to reach the public network. Therefore Federation will not work without a proxy. Some proxies require authentication.

Reply:
Thanks for the feedback, @micbar. Did you accidentally approve the PR?

Reply:
Can we talk about this on Friday?

Reply:
ok

Reply:
Actually, can you clarify what you mean, here?

Reply:
> I would like to have it better explained. There are oc instances that are not connected to the internet. They have no possibility to reach the public network. Therefore Federation will not work without a proxy. Some proxies require authentication.

I'm just seeking to find out how you want the explanation improved.

Reply:
@micbar, I took your first comment, reworked it a little, and integrated it into the required section.